packages/MonorepoBuilder/src/FileSystem/JsonFileManager.php

```diff
@@ -56,6 +56,6 @@ public function saveJsonWithFilePath(array $json, string $filePath): void
      */
     private function encodeJsonToFileContent(array $json): string
     {
-        return Json::encode($json, Json::PRETTY) . PHP_EOL;
+        return Json::encode($json, Json::PRETTY) . "\n";
```

Comment:
Why this change? The constant is system specific and also standard in the code.

Reply:
That exactly the reason I changed it. Json::encode uses `\n` regardless of the OS, as such the tests which compare file contents will fail on the last line of the file, as it is `\r\n` under Windows.